docs/content/how-to/configure-machines.md

```diff
@@ -0,0 +1,97 @@
+# Configuring Machines in HyperShift
+
+In standalone OpenShift, machine configuration is managed via resources like MachineConfig, KubeletConfig, and ContainerRuntimeConfig inside the cluster, and then applied to a set of machines via a MachineConfigPool. In HyperShift clusters, machine configuration is not managed from within a hosted cluster, but rather via the NodePool API outside the cluster. 
```

Comment:
may want to include this section into https://hypershift-docs.netlify.app/how-to/automated-machine-management/

Reply:
Thx! done